iam_units/data/emissions/AR4GWP100.txt

```diff
@@ -59,7 +59,6 @@
     a_HFE347pcf2 = 580.0
     a_HFE356pcc3 = 110.0
     a_HFE4310pccc124 = 1870.0
-    a_HFE449sl = 297.0
```

Comment:
Not hugely important but: does globalwarmingpotentials provide a change log that would explain deletions such as this one? I checked https://github.com/openclimatedata/globalwarmingpotentials but did not see one.

Reply:
See https://github.com/openclimatedata/globalwarmingpotentials/pull/14
There is no dedicated way of tracking changes beyond looking at issues/PRs and `git blame` - but I also don't think that this would be necessary for "less important" species.